### ISSUE TITLE
KeystrokeSequence element not found

### ISSUE THREAD

**Issue by pkozelka** [sru-systems/rust-kpdb#3]
I am using [KeepassX 2.0.3](https://www.keepassx.org/) to store my secrets, and would like to access it with this library (using current version `0.4.0`)

I wrote this code to test it out:

```rust
extern crate kpdb;

const PASSWORDFILE: &'static str = "...";
const KEYFILE: &'static str = "...";
const DBFILE: &'static str = "...";

use kpdb::{CompositeKey, Database, KeyFile};

fn main() {
    let password = std::fs::read_to_string(PASSWORDFILE).unwrap();
    let keyfile = &mut std::fs::File::open(KEYFILE).unwrap();
    let rdr = &mut std::fs::File::open(DBFILE).unwrap();
    let key = &CompositeKey::from_both(password,KeyFile::open(keyfile).unwrap());
    let db = Database::open(rdr, key).unwrap();
    for entry in db.root_group.entries {
        println!("{:?}", entry.uuid);
    }
}
```

Executing this code results in panicking on the line with `Database::open`:

```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: XmlError("4051:7 KeystrokeSequence element not found")', src/libcore/result.rs:1165:5
```

Am I doing anything wrong, or is my database format just unsupported?
Any chance to add support for this?

**Comment by mrijkeboer** [comment]
Hi @pkozelka I've merged your pull request (#4) and pushed a new version (0.4.1) to crates.io.

**Comment by mrijkeboer** [comment]
This issue should be fixed in 0.4.1.
